frontend/sass/_forms.scss

```diff
@@ -51,3 +57,12 @@ select {
     display: inline;
   }
 }
+
+.form-control-mono {
+  font-family: monospace,monospace;
```

Comment:
Does `monospace` need to be listed twice? (honest question, since it wouldn't surprise me if that was some css/browser-specific thing)

Reply:
Nope, good catch... thanks @jseppi